plugins/coverbrowser.koplugin/main.lua

```diff
@@ -0,0 +1,353 @@
+                    end,
+                    separator = true,
+                },
+                -- Plug the same choices for History here as a submenu
+                -- (Any other suitable place for that ?)
+                {
+                    separator = true,
+                    text = _("History display mode"),
+                    sub_item_table = {
+                        {
+                            text = _("Classic (filename only)"),
+                            checked_func = function() return not self.history_display_mode end,
+                            callback = function()
+                               self:setupHistoryDisplayMode("classic")
+                            end,
+                        },
+                        {
+                            text = _("Mosaic with cover images"),
+                            checked_func = function() return self.history_display_mode == "mosaic_image" end,
+                            callback = function()
+                               self:setupHistoryDisplayMode("mosaic_image")
+                            end,
+                        },
+                        {
+                            text = _("Mosaic with text covers"),
+                            checked_func = function() return self.history_display_mode == "mosaic_text" end,
+                            callback = function()
+                               self:setupHistoryDisplayMode("mosaic_text")
+                            end,
+                        },
+                        {
+                            text = _("List with image and metadata"),
+                            checked_func = function() return self.history_display_mode == "list_image_meta" end,
+                            callback = function()
+                               self:setupHistoryDisplayMode("list_image_meta")
+                            end,
+                        },
+                        {
+                            text = _("List with metadata, no image"),
+                            checked_func = function() return self.history_display_mode == "list_only_meta" end,
+                            callback = function()
+                               self:setupHistoryDisplayMode("list_only_meta")
+                            end,
+                        },
+                        {
+                            text = _("List with image and filename"),
+                            checked_func = function() return self.history_display_mode == "list_image_filename" end,
+                            callback = function()
+                               self:setupHistoryDisplayMode("list_image_filename")
+                            end,
+                            separator = true,
+                        },
+                    },
+                },
+                {
+                    text = _("Book infos cache maintenance"),
```

Comment:
`Book info cache maintenance`

Info/information is a mass noun like water, sand or electricity. You can't count it or pluralize it.

Also it might be better to call it *management* instead of *maintenance*?